test/fixtures/app/gulpfile.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+var gulp = require('gulp');
+var clean = require('gulp-rimraf');
+var statici18n = require('../../../');
```

Comment:
This is the test fixture importing the plugin via [index.js](https://github.com/yola/gulp-static-i18n/blob/9aba247446b58e14b694a56ce57d9d1735e12a41/index.js).